assets/js/elements-utils.js

```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
         node.removeChild( node.firstChild );
     }
 }
+
+export function makeCopyAllBadgesButtonElement(badgesAsMarkdown) {
```

Comment:
this method doesn't seem like a utils to me, but it contains domain logic and in my opinion it should stay inside the file app.js! Such as the method `renderBadgeContainer()`. no?

Reply:
:+1: